test/common.py

```diff
@@ -65,6 +81,22 @@ def pytest_runtest_setup(item):
             "Testing requiring observation disabled. Use --obs-enable flag to enable"
         )
 
+    # Handle LVDS tests
+    lvds = item.config.getoption("--lvds")
+    marks = [mark.name for mark in item.iter_markers()]
+    if not lvds and "lvds_test" in marks:
+        pytest.skip(
+            "LVDS testing disabled. Use --lvds flag to enable"
+        )
+
+    # Handle CMOS tests
+    cmos = item.config.getoption("--cmos")
+    marks = [mark.name for mark in item.iter_markers()]
+    if not cmos and "cmos_test" in marks:
+        pytest.skip(
+            "CMOS testing disabled. Use --cmos flag to enable"
+        )
+
```

Comment:
Can you add a check to make sure CMOS+LVDS are not enabled together.

Reply:
I now added one Travis, it's in 6c598a1, though I haven't had the chance to test it in the pipeline with an updated command in the CLI on jenkins-shared-lib. I ought to manually add either the --cmos or the --lvds flag and run the pipeline to check if it does work, but I don't have any access on the repo. Can you please check my code if it works as we intend it?